documentation/docs/overview.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: overview
+title: Overview
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Reveal is a JavaScript library for visualizing big 3D models on the web. It supports complex CAD models and large point clouds. The library supports 3D model hosted by [Cognite Data Fusion (CDF)](https://docs.cognite.com/cdf/) which supports [ingestion of a variety of 3D formats](https://docs.cognite.com/cdf/3d/). If you do not have access you can use models provided through the [Open Industrial Data project](https://openindustrialdata.com/) for testing.
+
+To get started using Reveal, see the [Getting started guide](./getting-started). If you have previously used the previous version of Reveal,[`@cognite/3d-viewer`](https://www.npmjs.com/package/@cognite/3d-viewer), and want to migrate to this package, please see the [Migration guide](./migration-guide). For details about the installation procedure and how to use Reveal in certain environments, see the [Installation guide](./installation).
```

Comment:
I guess it's better to use full file names with extensions. It might affect generated links. See my PR related to docs fixes  and https://v2.docusaurus.io/docs/versioning/#linking-files